### PR TITLE
switch intigration tests to launch irida via gradle

### PR DESCRIPTION
**Pull request by JeffreyThiessen**
## Description of changes
integration tests launch irida with gradle to match https://github.com/phac-nml/irida/pull/1319

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ Running the unittests can be done with the command:
 
 To run integration tests You will need to download and install chromedriver http://chromedriver.chromium.org/downloads
 
+You will need Gradle installed to run IRIDA
```
Comment (ericenns) [inline README.md:70]: You don't actually need gradle installed as the `./gradlew` will download it and then run it.

File: iridauploader/tests_integration/integration_data_setup.py [hunk 2]
```diff
@@ -83,6 +81,9 @@ def __init__(self, base_url, user, password, branch, db_host, db_port):
         self.REPO_PATH = path.join(self.PATH_TO_MODULE, "repos")
         self.IRIDA_PATH = path.join(self.REPO_PATH, "irida")
 
+        # Handles the irida subprocess
+        self.irida_subprocess = None
```
Comment (ericenns) [inline iridauploader/tests_integration/integration_data_setup.py:85]: this is just initializing the attribute so that you can use it in `install_irida`?

**Review by ericenns — CHANGES_REQUESTED** [review]
Just a few minor comments

File: iridauploader/tests_integration/integration_data_setup.py (hunk 2 above)
Comment (JeffreyThiessen) [reply iridauploader/tests_integration/integration_data_setup.py:85]: I init it there to be used in `run_irida` and `stop_irida`, In python it's good practice to init an attribute to `None` in the `__init__` when it's used in multiple functions. You can also do it as a class property, but depending on how it's used the IDE complains sometimes.

File: README.md (hunk 1 above)
Comment (JeffreyThiessen) [reply README.md]: Fixed.

**Pull request merged**